NewForce-Cohort-8/nutshell-soul-2-seoul
# Ht eventlist

# Description
 This branch includes a functioning "event list" form where users can enter important information about an event and it will log to the browser as a card. Users can add and delete cards. 

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions

Fill out Event List form and confirm that data renders to browser. Add and delete event list posts.

# Checklist:
